src/auth/auth.service.ts

```diff
@@ -62,6 +65,25 @@ export class AuthService {
       };
       // Return the user info who has been created when logging
       return newUserInfo;
+    } else if (!user.isActivated) {
+      await this.userService.updateUserById({
+        userId: user._id,
+        googleId: sub,
+        isActivated: true,
+      });
+      // get access token and refresh token
+      const tokens = await this.getTokens(user._id, user.email);
+      await this.updateRtHash(user._id, tokens.refreshToken);
+      const updatedUserInfo: ReturnUserInfo = {
+        userId: user._id,
+        googleId: sub,
+        email: email,
+        firstName: user.firstName,
+        lastName: user.lastName,
+        tokens,
+        needConnection: false,
+      };
+      return updatedUserInfo;
```

Comment:
That's a good point! I'll extract them if I use them elsewhere in the future! However, I guess it is unnecessary to extract them since these codes are only used in one method for now. What do you think of that?